app/monitoring/SentryLogging.scala

```diff
@@ -1,31 +1,28 @@
 package monitoring
 
-import ch.qos.logback.classic.{Logger, LoggerContext}
-import com.getsentry.raven.RavenFactory
-import com.getsentry.raven.dsn.Dsn
-import com.getsentry.raven.logback.SentryAppender
-import org.slf4j.Logger.ROOT_LOGGER_NAME
-import org.slf4j.LoggerFactory
-import com.gu.support.config.Stage
-import app.BuildInfo
+import monitoring.SafeLogger._
 
-object SentryLogging {
-  val AllMDCTags = Seq()
-}
+import config.Configuration
+import io.sentry.Sentry
 
-class SentryLogging(dsnConfig: String, stage: Stage) {
-  val dsn = new Dsn(dsnConfig)
-  SafeLogger.info(s"Initialising Sentry logging for ${dsn.getHost}")
-  val tags = Map("stage" -> stage.toString) ++ BuildInfo.toMap
-  val tagsString = tags.map { case (k, v) => s"$k:$v" }.mkString(",")
-  val sentryAppender = new SentryAppender(RavenFactory.ravenInstance(dsn)) {
-    addFilter(SentryFilters.errorLevelFilter)
-    addFilter(SentryFilters.piiFilter)
-    setTags(tagsString)
-    setRelease(BuildInfo.gitCommitId)
-    setExtraTags(SentryLogging.AllMDCTags.mkString(","))
-    setContext(LoggerFactory.getILoggerFactory.asInstanceOf[LoggerContext])
+import scala.collection.JavaConverters._
+import scala.util.{Failure, Success, Try}
+
+object SentryLogging {
+  def init(config: Configuration) {
+    config.sentryDsn match {
+      case None => SafeLogger.warn("No Sentry logging configured (OK for dev)")
+      case Some(sentryDSN) =>
+        SafeLogger.info(s"Initialising Sentry logging with $sentryDSN")
+        Try {
+          val sentryClient = Sentry.init(sentryDSN)
```

Comment:
We were setting a release on the SentryAppender, but we're sending this information to Sentry as a tag anyway, so is this this needed? Also, I'm not covering setContext on the Sentry appender elsewhere -- does anyone know what this is needed for and how to do it in sentry-logback?